### DISCUSSION TITLE
Only FEC activation

### DISCUSSION THREAD

**Discussion by kubut007** [Mr-MIBonk/M.I.B._More-Incredible-Bash#52] [Q&A] [answered]
Hello,
I need only added FEC codes to my unit, I dont want remove CP, because it was officially unlocked by ODIS.
I have firmware MHI2_ER_VWG13_P4521 MU1367.
Please could you help me??
Thank you for any info.

**Comment by harman-f** [comment, marked as answer]
You need to patch, otherwise new FECs will not be accepted.
This will **not** break your original unlock with ODIS.
Both can run inparallel fine.

**Reply by kubut007** [reply]
Thank you for reply. 
Could you provide the Only FEC Patch File or can I patched it automatically by M.I.B, please?
I am a bit confused, because there is written "CP + FEC + VCRN removed" in Info.txt with the Pach File for M.I.B.

[Info.txt](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9933144/Info.txt)

**Reply by harman-f** [reply]
CP check will be removed by the patch, but this will not effect your ODIS CP removal.
It will stay valid meaning even after restoring to stock it will continue to work.

Just patch, there will be no issues with your ODIS CP ;)

**Reply by kubut007** [reply]
The M.I.B. Tool works exactly as you wrote. 
I patched it today and everything is right. 
I only removed the SVM error and there are No DTCs on diagnostic. 
Great Tool, just what I needed.
Thank you very much
